tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "outDir": "./lib",
     "rootDir": "./src",
     "paths": {
-    }
+    },
+    "jsx": "react-native",
+    "skipLibCheck": true
```

Comment:
do we need skipLibCheck?

Reply:
There are issues with RN packages because they import `@types/node` and that causes `Duplicate identifier` TS issues. Official RN app also uses `skipLibCheck: true`: https://github.com/tsconfig/bases/blob/main/bases/react-native.json#L22